reference-adapter/ifix-department-entity-service/src/main/resources/db/migration/main/V202211131027__department_entity_ddl.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS public.department_entity
+(
+    id character varying(255) COLLATE pg_catalog."default" NOT NULL,
+    children character varying(5000) COLLATE pg_catalog."default",
+    code character varying(64) COLLATE pg_catalog."default",
+    created_by character varying(255) COLLATE pg_catalog."default",
+    created_time bigint,
+    department_id character varying(64) COLLATE pg_catalog."default",
```

Comment:
What is the purpose of `COLLATE pg_catalog."default"`? 
If it isn't serving any significant domain-specific purpose, you should keep it simple and use the pattern followed in other DIGIT-Dev services.

Reply:
ok